assets/behaviors/creatures/combinedCritter.behavior

```diff
@@ -0,0 +1,107 @@
+{
+selector: [
+  {
+    guard: {
+
+      // Territory would to be implemented as Component
+      // Implementation of territorial Critter
+      componentPresent: "Behaviours:Territory",
```

Comment:
I don't think the `TerritoryComponent` exists in the Behaviors module.

The closest thing we have is [TerritoryDistance](https://github.com/Terasology/Behaviors/blob/develop/src/main/java/org/terasology/module/behaviors/components/TerritoryDistance.java), a component with two fields `distance` and `location`.

Therefore, the referenced component should probably be 
```suggestion
      componentPresent: "Behaviours:TerritoryDistance",
```

and the referenced field below cannot be `Territory` (as it does not exist) but either `distance` or `location`. I suspect that we can check whether the location is set.